core/src/main/java/io/grpc/internal/HedgingPolicy.java

```diff
@@ -75,6 +71,8 @@ public String toString() {
         .toString();
   }
 
+  // TODO(zdapeng): delete this because HedgingPolicy will be always available prior to starting
+  //  RetriableStream.
```

Comment:
Is there anything blocking this to be deleted? Otherwise, we can just get rid of this now.

Reply:
Some tests need change accordingly, so I decided to change it in next PR.